builder/azure/chroot/diskattacher.go

```diff
@@ -123,6 +123,8 @@ func (da *diskAttacher) AttachDisk(ctx context.Context, diskID string) (int32, e
 
 	// disk was not found on VM, go and actually attach it
 
+	// TODO This assignment looks like it would do nothing, consider removing it
```

Comment:
The variable is being used after the for loop, which is while it is being defined before the label. The logic is a little confusing. I think this could benefit from a refactor. 
```suggestion
	// TODO This logic is a little hard to read; refactor to remove the label. 
```